app/controllers/workspace/teams_controller.rb

```diff
@@ -1,19 +1,21 @@
 module Workspace
   class TeamsController < WorkspaceController
+    skip_verify_authorized
+
     def index
       @users = authorized_scope(User, type: :relation).all
     end
 
     def new_modal
-      @user = User.new
+      @user = authorized_scope(User, type: :relation).new
```

Comment:
The idea was that if we were using `authorize!` and `authorized_scope` every time we do a query, it would be easy to spot the places where we missed them, and prevent data leaking out.

TBH I really hadn't though about the performance impact, but I'd keep it like that if helps to keep the data secure.